certs/sp/demo_sp.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
not to be too pedantic, but the service provider is micropurchase. The identity idp app is the identity provider.